### ISSUE TITLE
Problem running ESMF tools in parallel on cheyenne

### ISSUE THREAD

**Issue by mnlevy1981** [ESMCI/cime#3028]
Following the machine updates from March 5, 2019, the ESMF binaries compiled with the mpt library on Cheyenne no longer work (they were compiled against mpt/2.15 and that module was removed). Attempting to run in parallel results in the error

```
cmd is mpirun -np 1 ESMF_RegridWeightGen --ignore_unmapped -m bilinear -w map_T62_TO_gx3v7_120309.nc_blin.190307.nc -s /glade/p/cesmdata/cseg/mapping/grids/T62_040121.nc -d /glade/p/cesmdata/cseg/mapping/grids/gx3v7_120309.nc

MPT ERROR: Cannot run application.
	A version 2.19 libxmpi is being used with version 2.15 libmpi. Ensure
	that the same version of MPT is installed or loaded via environment
	modules on the host running mpirun and the hosts running the
	application.
	(HPE MPT 2.19  12/07/18 05:32:16)
```

(similar errors occur with `mpt/2.16` and `mpt/2.18`). I've asked CISL to rebuild ESMF modules, and will update `create_ESMF_map.sh` when this has been resolved. In the meantime, `create_ESMF_map.sh --serial` still works.

**Comment by billsacks** [comment]
@mnlevy1981 should this be fixed now?

Also, does create_ESMF_map.sh need to be updated to use esmf 7.1.0r rather than 7.0.0?

**Comment by mnlevy1981** [comment]
To answer your second question, the mapping tools were updated to 7.1.0r a while ago (as soon as we recognized the bug in 7.0.0). I don't think CISL has re-built the `esmf-7.1.0r-ncdfio-mpi-O` module with a working version of `mpt` yet, so I believe we're still stuck using `--serial` for the time being. I emailed `cisl-help` requesting they rebuilt the parallel modules around the same time I opened this issue ticket, so hopefully they'll do it soon.
